sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -28,8 +28,9 @@ class PhoneNumberIdentifier(object):
     :param value: Value to initialize PhoneNumberIdentifier.
     :type value: str
     """
-    def __init__(self, phone_number):
+    def __init__(self, phone_number, identifier=None):
```

Comment:
Are you missing `identifier` in the type metadata?

Please copy over the comments from C# to call out that identifier is the **full** id and that phone_number is in E.164 format.

Also the type metadata still calls it `value`, it's `phone_number` now.

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Common/src/PhoneNumberIdentifier.cs